papaparse.js

```diff
@@ -1285,21 +1285,25 @@
 							continue;
 						}
 
-						// Closing quote followed by delimiter
-						if (input[quoteSearch+1] === delim)
+						var spacesBetweenQuoteAndDelimiter = extraSpacesLengthAfterQuote(nextDelim);
```

Comment:
Probably we can find a shorter name. Why not extraSpaces?

Reply:
Agree. was just trying to be specific.